nipype/interfaces/ants/legacy.py

```diff
@@ -94,16 +94,20 @@ class antsIntroduction(ANTSCommand):
 
     def _list_outputs(self):
         outputs = self._outputs().get()
+        transmodel = self.inputs.transformation_model
+
+        # When transform is set as 'RI'/'RA', no wrap fields will be outputed
+        if isdefined(transmodel) and transmodel not in ['RI', 'RA']:
```

Comment:
@satra Thx for pointing out. You are right about it. I just made the according change and locally tested it with real nii images.